test/EFCore.PG.FunctionalTests/TestUtilities/Xunit/MinimumPostgresVersionFactAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Xunit;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities.Xunit
+{
+    public class MinimumPostgresVersionFactAttribute : FactAttribute
+    {
+        private readonly int _major;
+        private readonly int _minor;
+
+        public MinimumPostgresVersionFactAttribute(int major, int minor)
+        {
+            _major = major;
+            _minor = minor;
+        }
```

Comment:
Forgot that it's possible to have a non-primitive property, so this can be simplified to:

```csharp
public class MinimumPostgresVersionFactAttribute : FactAttribute
{
    public Version Version { get; }

    public MinimumPostgresVersionFactAttribute(int major, int minor) =>
        Version = new Version(major, minor);
}
```